src/dodal/beamlines/i22.py

```diff
@@ -77,16 +121,17 @@ def synchrotron(
     )
 
 
+# d12 at p38
 def waxs(
     wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
 ) -> PilatusDetector:
     return device_instantiation(
-        NXSasPilatus,
+        AravisDetector if IS_LAB else NXSasPilatus,
         "waxs",
-        "-EA-PILAT-03:",
+        "-DI-DCAM-04:" if IS_LAB else "-EA-PILAT-03:",
         wait_for_connection,
         fake_with_ophyd_sim,
-        drv_suffix="CAM:",
+        drv_suffix="DET:" if IS_LAB else "CAM:",
         hdf_suffix="HDF5:",
         metadata_holder=NXSasMetadataHolder(
```

Comment:
This metadata isn't valid for the lab device, and could make analysis wrong (e.g. pixel size, distance used to calculate angles). This is part of why I'd prefer to see structured as a single ternary rather than multiple ternaries inside the device_instantiation function